.flake8

```diff
@@ -0,0 +1,17 @@
+[flake8]
+extend-ignore:
+    E114,
+    E115,
+    E116,
+    E201,
+    E202,
+    E203,
+    E204,
+    E231,
+    E265,
+    E266,
+    E303,
+    E402,
+    E501,
```

Comment:
I think we should add, locally collects files differently to the GA 
to make it ignore `__init__.py` and `setup.py` files and to stop it searching docs, when run locally, we can also add to tests it to ignore this as well
```
exclude =
    *__init__.py
    *setup.py
    docs
 ```